interfaces/builtin/kvm_test.go

```diff
@@ -57,8 +66,23 @@ slots:
 `
 
 func (s *kvmInterfaceSuite) SetUpTest(c *C) {
+	s.BaseTest.SetUpTest(c)
+
 	s.plug, s.plugInfo = MockConnectedPlug(c, kvmConsumerYaml, nil, "kvm")
 	s.slot, s.slotInfo = MockConnectedSlot(c, kvmCoreYaml, nil, "kvm")
+
+	// Need to Mock output of /proc/cpuinfo
+	s.tmpdir = c.MkDir()
+	dirs.SetRootDir(s.tmpdir)
```

Comment:
Please restore original root directory in `TearDownTest()`

Reply:
Done